themes/odin-theme/layouts/partials/meta_cards.html

```diff
@@ -6,14 +6,14 @@
   <meta property="og:image" content="{{ . | absURL }}" />
   <meta property="og:image" content="{{ . | absURL }}" />
   {{ end }}{{ else }}
-  <meta property="og:image" content="{{ "/images/logo-slim.png" | absURL }}" />
+  <meta property="og:image" content='{{ "/images/logo-slim.png" | absURL }}' />
```

Comment:
Why are the tags here changed to `'` from `"`?

Reply:
because it was throwing an error in VS Code.